conf/messages

```diff
@@ -42,6 +42,8 @@ upload.selectFolder=Select a folder
 upload.fileExtensionTitle=Which file extensions are not permitted?
 upload.fileExtensionSummary=Cannot upload the following file extensions: .zip, thumbs.db, .exe
 upload.continueLink=Continue
+upload.error.title=There is a problem
```

Comment:
In case anyone's wondering about the wording, we've got a card to come back and replace the placeholder error messages once we've done some work on the content design: https://national-archives.atlassian.net/browse/TDR-36

Reply:
Added a comment to the file to say placeholder wording